config.docker.yml

```diff
@@ -0,0 +1,7 @@
+public_dir: ./public
+packages_paths:
```

Comment:
I'm torn if we should call it `package_paths`?

Reply:
Renamed.